libs/logger/src/logLevel-common-config.ts

```diff
@@ -9,15 +11,16 @@ export function getCurrentLogLevel(): string {
 }
 
 export function getPinoTransport() {
-  if (process.env.PRETTY === 'true') {
+  if (/^true|compact/.test(process.env?.PRETTY)) {
     return {
       target: 'pino-pretty',
       options: {
-        colorize: true,
-        colorizeObjects: true,
+        colorize: isColorSupported,
+        colorizeObjects: isColorSupported,
```

Comment:
We need to change how we are passing in the value.

See test logs:
`ERROR in /home/runner/work/gateway/gateway/libs/logger/src/logLevel-common-config.ts
./libs/logger/src/logLevel-common-config.ts 18:18-34
[tsl] ERROR in /home/runner/work/gateway/gateway/libs/logger/src/logLevel-common-config.ts(18,19)
      TS2693: 'isColorSupported' only refers to a type, but is being used as a value here.`

Reply:
The `pino-pretty` github page shows using the value like this: https://github.com/pinojs/pino-pretty?tab=readme-ov-file#checking-color-support-in-tty

Alternatives might be requiring the `colorette` library directly: `import { isColorSupported } from 'colorette';` 

or using `const colorDepth = process.stdout.getColorDepth();` and converting the number returned into a boolean.

Reply:
I'm pretty sure it's a bug in pino-pretty's Typescript definitions file; I've pushed a workaround that changes the `import` to `require` which seems to work.